rd_ui/app/views/dashboard.html

```diff
@@ -42,7 +42,7 @@ <h3 class="panel-title">
                     </h3>
                 </div>
 
-                <visualization-renderer visualization="widget.visualization" query-result="queryResult"></visualization-renderer class="panel-body">
+                <visualization-renderer visualization="widget.visualization" query-result="queryResult" hide-filters="dashboard.dashboard_filters_enabled"></visualization-renderer class="panel-body">
```

Comment:
Basically showing a "**\<Name\>:** \<Value\>" pair in text for each dashboard-wide filter applying to a widget in place of where the inputs normally go.  It's really the styling of the thing that's cumbersome when dashboard-wide filters are used, and plain text is easier to style concisely and compactly than the input elements that are used normally.